output/cloud/expv2/output.go

```diff
@@ -231,3 +304,24 @@ func (o *Output) handleFlushError(err error) {
 		}
 	})
 }
+
+func (o *Output) tracingEnabled() bool {
+	// TODO(lukasz): Check if the `tracing` module is imported.
+	//
+	// We want to check if the `tracing` module was imported. If
+	// it wasn't, we don't need to collect any data related to tracing
+	// as there won't be any.
+	//
+	// We currently don't have a way to check if the tracing module
+	// was imported by the user.
```

Comment:
Based on the previous discussion it is not required anymore, right?
```suggestion
```

Reply:
Removed in [19fa719](https://github.com/grafana/k6/pull/3100/commits/19fa7196d032785f5c5a52eb6c83755f98e8e090).